src/configurations/csbc-pson/synapseConfigs/onPointClick.ts

```diff
@@ -10,6 +49,11 @@ export const onPointClick = ({
   if (typeUpperCase === 'Grants' || typeUpperCase === 'Projects') {
     facet = 'consortium'
   }
+  const encodedQuery = generateEncodedQueryForURL(
+    typeUpperCase,
+    facet,
+    facetValue,
+  )
   // @ts-ignore
-  window.location = `/Explore/${typeUpperCase}?facet=${facet}&facetValue=${facetValue}`
+  window.location = `/Explore/${typeUpperCase}?QueryWrapper0=${encodedQuery}`
```

Comment:
this will work if there is a single QueryWrapper on the page or if there is more than one it would work with the first.